### ISSUE TITLE
Issue with Postgres test cases RMLTC0005a-PostgreSQL and RMLTC0005b-PostgreSQL

### ISSUE THREAD

**Issue by marioscrock** [kg-construct/rml-core#123]
The table "IOUs" is not found since the `.sql` file used for database initialization does not put the table name between double quotes and Postgres by default initializes the table as "ious" (lowercase). 

I solved this issue by forcing a lowercase conversion for table names extracted from RML if the source is a Postgres DB. However, I don't know if this is a good assumption or if it would be better to update the test case.

**Comment by DylanVanAssche** [comment]
Interesting! I have seen this problem like _a lot_. IMO, we should even define this as part of the [rml-io-registry](https://github.com/kg-construct/rml-io-registry) where we define for each reference formulation 'special' handling. We cannot change the behavior of various RDBMS vendors so we have to define how to handle these cases I guess?

**Comment by dachafra** [comment]
I agree with you @DylanVanAssche, if you think it is more related to the rml-io-registry, we can transfer the issue to that repository.
